R/process_lookup_gpprac.R

```diff
@@ -4,21 +4,22 @@
 #' gpprac lookup, it will return the final data
 #' but also write this out as a zsav and rds.
 #'
+#' @param file_path Link to phs open dataset
```

Comment:
I don't think like this - I see why you've done it but, this isn't a link to the open data (arguably that would be `"gp-practice-contact-details-and-list-sizes"`)... Do we need to have this as a target? What do we gain from having it as a target?

I think we can just leave it as it was before, so the only parameter (and therefore target) is `write_to_disk`. Happy to have a chat though!

Reply:
TODO - split phs open data out into its own function

Reply:
I've split this out into its own function. Ready for re-review